app/src/main/kotlin/at/bitfire/davdroid/ui/intro/BatteryOptimizationsPage.kt

```diff
@@ -178,8 +146,11 @@ class BatteryOptimizationsPage: IntroPage {
                 context.getSystemService<PowerManager>()!!.isIgnoringBatteryOptimizations(BuildConfig.APPLICATION_ID)
         }
 
-        val shouldBeExempted = MutableLiveData<Boolean>()
-        val isExempted = MutableLiveData<Boolean>()
+        var shouldBeExempted by mutableStateOf(true)
+            private set
+        var isExempted by mutableStateOf(false)
```

Comment:
These variables should be in `data class UiState` created as `var uiState by mutableStateOf(UiState())` and be updated with `uiState = uiState.copy(isExempted = ...)` etc.